Frontend/src/clients/UserClient.ts

```diff
@@ -174,4 +174,37 @@ export class UserClient {
             return "";
         }
     };
+
+    async changeProfilePicture(imageUrl: string | null) {
+        try {
+
+            // Getting a response Body back so i can display it instantly
+            const response = await fetch('https://localhost:44328/User/upload-profile-picture', {
```

Comment:
du kannst statt den request zu benutzen einfach den base64-string in den image-tag reinhauen, dann wird das bild auch angezeigt.

So wie dus jetzt machst wird das bild nämlich 2 mal geuploaded und 2 mal gespeichert, was den image folder unnötig aufmüllt